lib/content-services/src/lib/category/categories-management/categories-management.component.spec.ts

```diff
@@ -167,44 +167,17 @@ describe('CategoriesManagementComponent', () => {
                 component.categoryNameControlVisible = true;
                 fixture.detectChanges();
             });
-            it('should be hidden initially', () => {
+            it('should not hide category name control when categoryNameControlVisible is false', () => {
```

Comment:
does not make sense, you are changing the test behaviour, also the test seems confusing: category name is not visible, but you test it's visible?